### ISSUE TITLE
error about  vista#parser#ctags#FromExtendedRaw()

### ISSUE THREAD

**Issue by Freed-Wu** [liuchengxu/vista.vim#253]
once enter a new window, this message will appear:
```
|| Error detected while processing function
|| <SNR>121_close_cb[5]
|| vista#parser#ctags#FromExtendedRaw[9]
|| E684: list index out of range: 1
|| E15: Invalid expression: items[1]
```

OS: WindowsOS10
gvim:8.1
ctags:5.8
vista:latest

vista info:
```
    Current FileType: vim
Avaliable Executives: ['ctags']
    Global Variables:
    let g:vista#executive#ctags#support_json_format = 0
    let g:vista#executives = ['ale', 'coc', 'ctags', 'lcn', 'nvim_lsp', 'vim_lsc', 'vim_lsp']
    let g:vista#extensions = ['markdown', 'rst']
    let g:vista#finders = ['fzf', 'skim']
    let g:vista_close_on_jump = 1
    let g:vista_fold_toggle_icons = ['', '']
    let g:vista_sidebar_width = 41
```

i notice this error comes from
```
function! vista#parser#ctags#FromExtendedRaw(line, container) abort
  if a:line =~# '^!_TAG'
    return
  endif
  let items = split(a:line, '\t')

  let line = {}

  let line.name = items[0]
  let line.tagfile = items[1]
```

perhaps should add some code 
```
if len(item)>0
  let line.name = items[0]
  let line.tagfile = items[1]
endif
```
to avoid this error?
Thanks!

**Comment by liuchengxu** [comment]
Please fill the issue template completely, especially providing the minimal vimrc and sample file for ctags related issue, otherwise it's not reproducible.

**Comment by Freed-Wu** [comment]
test.vim
```
set nocompatible
set runtimepath+=$GITHUBWORKSPACE/liuchengxu/vista.vim
```

```
gvim -u C:\the\path\to\test.vim
```

then `:Vista`. this error will occur.

**Comment by liuchengxu** [comment]
The test file is required for the ctags issue.

**Comment by BlueDrink9** [comment]
I'm having this issue as well. Seeing as it also appears in liuchengxu/vim-clap/issues/372, but you can't reproduce it personally (meaning it's common enough, but hard to track down), would you consider including a fix just in case?

That function seems to assume a lot about the number of tab-separated fields in a:line. I think some check is needed at least, because clearly a:line isn't always meeting those expectations.

I couldn't fix the error with the above suggested fix, but I could with this patch: #268
